Makefile

```diff
@@ -149,8 +149,14 @@ df.key:
 	openssl genrsa -des3 -passout pass:x -out df.pass.key 2048
 	openssl rsa -passin pass:x -in df.pass.key -out df.key
 	rm df.pass.key
+.PHONY: df.key
 
 df.crt: df.key
 	openssl req -new -key df.key -out df.csr
 	openssl x509 -req -sha256 -days 365 -in df.csr -signkey df.key -out df.crt
 	rm df.csr
+.PHONY: df.crt
+
+fmt:
+	@gofmt -s -w .
+.PHONY: fmt
```

Comment:
I hope we could do more with command `make fmt`, not only just gofmt. We should add goimports or some other checks too.

Reply:
Thanks @yeya24 
I want to add `make update` command to do all the related work later, such as `go fmt`, `go vet` and so on.
But `make fmt` is also useful. In some cases, developer just need `go fmt` the code rather than do all checks. WDYT?

Reply:
Agree :smile: I think at last we can have a `make format` to do all format checking and linting.Then we can replace the checks in ci.